code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm

```diff
@@ -53,6 +53,18 @@
 		return 1
 	return 0
 
+/obj/item/organ/internal/body_egg/terror_eggs/Initialize(mapload)
+	. = ..()
+	GLOB.ts_infected_list += src
+
+
+/obj/item/organ/internal/body_egg/terror_eggs/Destroy()
+	GLOB.ts_infected_list -= src
+	return ..()
+
+/obj/item/organ/internal/body_egg/terror_eggs/on_owner_death()
+	GLOB.ts_infected_list -= src
```

Comment:
All of these `src` will be references to the eggs themselves, maybe it would be more helpful to have the list be the eggs' `owner` var instead? So admins could look at the list and see who is infected? I'll leave it up to you

Reply:
using owner doesn't seem to work so I'm just going to stick with src, I don't think its often admins will actually go through the MC panel and check globals too look for who is bitten anyways it should be fine